build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <PropertyGroup>
+    <BuildDepends>
```

Comment:
The default behavior sounds good. But for override, the target should be named something other than `Build` - `ReallyBuild`, `ForcedBuild`, `BuildIgnoringCache` etc.